apps/.storybook/preview.js

```diff
@@ -11,3 +11,4 @@ export const parameters = {
     },
   },
 };
+export const tags = ['autodocs'];
```

Comment:
Hmm, what is the cost of making the `preview.ts` version:

```typescript
import type { Preview } from '@storybook/react';

import $ from 'jquery';

//Stub jquery fileupload library function
$.fn.fileupload = () => {};
 
const preview: Preview = {
  parameters: {
    options: {
      storySort: {
        method: 'alphabetical',
        order: ['DesignSystem', 'templates', 'code-studio'],
      },
    },
  },
  tags: ['autodocs'],
};
 
export default preview;
```

Reply:
I just tried this but unfortunately there's a bug in `main.ts` which is fixed in typescript 5.5 (our version is 5.3). I'll circle back on this after we upgrade typescript.